angular/core/components/thread_page/activity_card/discussion_edited_item_controller.coffee

```diff
@@ -1,6 +1,6 @@
-angular.module('loomioApp').controller 'DiscussionEditedItemController', ($scope) ->
-  discussion = $scope.event.discussion()
-  version = $scope.event.version()
+angular.module('loomioApp').controller 'DiscussionEditedItemController', ($scope, Records) ->
+  version    = Records.versions.find($scope.event.eventable.id)
+  discussion = Records.discussions.find(version.discussionId)
```

Comment:
Actually, looking at this with fresh eyes, it should probably be

```
# event_model.coffee
eventable: ->
  @recordStore[@eventable.type].find(@eventable.id]
```
which I think we do with notifications.

Anyway this is a solid step up for now.